README.md

```diff
@@ -33,25 +86,172 @@ VS Code will generate a `launch.json` in your project with some default configur
+
+Click [here](https://reactnative.dev/docs/hermes) to learn more about Hermes.
+
+Debugging apps with Hermes enabled is currently experimental. Please, see [this issue](https://github.com/microsoft/vscode-react-native/issues/1073) for current known issues on Hermes support.
+
+To debug while using Hermes engine, please choose one of the following debug configurations:
+ - React Native (Hermes): Debug Android - Experimental
+
+```json
+{
+    "name": "Debug Android (Hermes) - Experimental",
+    "cwd": "${workspaceFolder}",
+    "type": "reactnativedirect",
+    "request": "launch",
+    "platform": "android"
+}
+```
+
+ - React Native (Hermes): Attach to Hermes application - Experimental
+
+```json
+{
+    "name": "Attach to Hermes application - Experimental",
+    "cwd": "${workspaceFolder}",
+    "type": "reactnativedirect",
+    "request": "attach"
+}
+```
+
+## iOS devices
+
+Debugging on an iOS device requires following manual steps:
+* Install [ios-deploy](https://www.npmjs.com/package/ios-deploy) `npm install -g ios-deploy`.
+* Install a valid iOS development certificate.
+* In your project's `launch.json` file set `target` to `device`. If you need to specify the exact device to run, you can set `target` to `device=<iOS_device_name>`, or you can also use `runArguments` property to specify a particular device to run on in case multiple devices are connected (e.g. `"runArguments": [ "--device", "My iPhone" ]`)
+* Choose the **Debug iOS** option from the "Configuration" dropdown and press F5.
+* Shake the device to open the development menu and select "Debug JS Remotely".
+
+## Customs scheme for iOS apps
+
+If you want to use a custom scheme for your application you can either pass it as part of the `runArguments` parameter arguments, or set the `scheme` configuration parameter as shown below:
+```js
+"runArguments": ["--scheme", "customScheme", ...]
+// or
+"runArguments": ["--scheme=customScheme", ...]
+// or
+"scheme" : "customScheme"
+```
+Please be aware, specifying the scheme value as a part of the `runArguments` parameter arguments will override the `scheme` configuration parameter value, if it set.
+
+## Expo applications
+
+To debug a project created using Expo or the create-react-native-app task, you can use embedded support for Expo.
+
+Your environment must meet the following prerequisites:
 
-Since version 0.60.2 [React Native supports Hermes engine for Android applications](https://facebook.github.io/react-native/blog/2019/07/17/hermes). Extension provides experimental support for debugging React Native Android applications with Hermes enabled. Please see [Debugging React Native apps with Hermes enabled](doc/debugging.md#debugging-react-native-apps-with-hermes-enabled) for more details.
+- Install the [Expo app](https://getexponent.com/) on the target device or emulator
+- Ensure that the react-native-cli is installed globally (`npm install -g react-native-cli`)
```

Comment:
```suggestion
- Ensure that the react-native-cli is installed globally (`npm install -g @react-native-community/cli`)
```

Reply:
I believe this whole section is outdated. Installing globally the RN CLI is not recommended anymore and the steps to get [Expo up and running nowadays seem to be different](https://reactnative.dev/docs/environment-setup):

> Assuming that you have Node 12 LTS or greater installed, you can use npm to install the Expo CLI command line utility:
> `npm install -g expo-cli`
> Then run the following commands to create a new React Native project called "AwesomeProject":
> ```
> expo init AwesomeProject`
> cd AwesomeProject
> npm start # you can also use: expo start
> ```
> This will start a development server for you.

@SounD120 @RedMickey can you please confirm and tell me if there's any other requirements than the ones in the docs?

Reply:
@molant Actually we might even don't need `expo-cli` at all for running Expo debugging scenarios since we install [`@expo/xdl`](https://www.npmjs.com/package/@expo/xdl) package inside the extension which is doing all the communication with Expo services. But it might be useful to init fresh Expo apps and to work with some of the extension commands. With respect to the React Native CLI - since [v0.13.0](https://github.com/microsoft/vscode-react-native/blob/master/CHANGELOG.md#0130) we can work with the locally installed CLI so the global one is not required.
@RedMickey correct me if I missed something.

Reply:
Yep, It's possible to debug an Expo app without having `expo-cli` installed. I tried to do this and was able to debug an Expo app without problems.
Also now we don't use global react-native CLI.

Reply:
@RedMickey @SounD120 do you mind suggesting new instructions for this section?

Thanks!

Reply:
Yes, it is better to update them

Reply:
@RedMickey @SounD120 what would be the new steps for this? I believe it's the last comment to get this merged 😊

Reply:
I think we'll finally merge this and do all enhancements in another PRs. It was here for too long because of problems with CI tests, but now they gone, and I think we are good to go :)

Reply:
SGTM!
Will you take care of merging? Something else you need me to do?

Reply:
I am waiting for the tests now, once they are passed I'll merge this.